lib/usdUfe/ufe/trf/Utils.h

```diff
@@ -24,48 +24,36 @@
 
 #include <ufe/path.h>
 
-PXR_NAMESPACE_USING_DIRECTIVE
-
 namespace USDUFE_NS_DEF {
 
 //------------------------------------------------------------------------------
 // Private helper functions
 //------------------------------------------------------------------------------
 
 //! Extended support for the xform operations.
-// TEMP (UsdUfe) - export this function so it can be used by mayaUsd/ufe
-USDUFE_PUBLIC
-UsdGeomXformCommonAPI convertToCompatibleCommonAPI(const UsdPrim& prim);
```

Comment:
These functions where exported so they could be used by Maya. The code that called them was moved to UsdUfe and these are only needed by the transform code. So moved file into trf folder and removed export.